src/status_im/constants.cljs

```diff
@@ -82,10 +82,15 @@
           (when config/mainnet-networks-enabled? mainnet-networks))))
 
 ;; adamb's status-cluster enode
-(def default-wnode {:name "Status mailserver"
-                    :address "enode://08d8eb6177b187049f6c97ed3f6c74fbbefb94c7ad10bafcaf4b65ce89c314dcfee0a8bc4e7a5b824dfa08b45b360cc78f34f0aff981f8386caa07652d2e601b@163.172.177.138:40404"})
+(def default-wnode "main")
 
-(def default-wnodes [default-wnode])
+(def default-wnodes
+  {"main" {:id      "main"
```

Comment:
I think key should be :main, and do we need the id in the value when it is already a map?

Reply:
Actually I see why, it's because of realm right?